packages/libs/hd-wallet/README.md

```diff
@@ -186,6 +321,55 @@ const decryptedSeed = kadenaDecrypt(password, seed);
 console.log(decryptedSeed); // Outputs the decrypted seed
 ```
 
+## Integration Guide for Developers
+
+### How to Use in Your Wallet Application
+
+If you’re building a wallet application, you can integrate these key generation
+and signing functionalities directly. Here’s a simple flow for implementation:
+
+1. **Generate a mnemonic phrase** and store it securely (e.g., show the user
+   their backup phrase).
+2. **Convert the mnemonic to a seed** and encrypt it with a password before
+   storing.
+3. **Generate key pairs as needed** using indices (e.g., index 0 for the primary
+   account, index 1 for a secondary account).
+4. **Sign transactions** with the private key or directly with the seed.
+5. **Retrieve public keys** when you need to display addresses or verify
+   signatures.
+
+### Key Derivation Path
+
+The Kadena wallet uses the standard derivation path:
```

Comment:
add link to "packages/libs/hd-wallet/docs/decisions/0001-use-slip10-for-private-key-generation.md"